### ISSUE TITLE
Player specific permissions for commands and client side features

### ISSUE THREAD

**Issue by damonmensch** [Lunatrius/Schematica#86]
It would be helpful if you enabled permission nodes for servers with schematica to limit who can use it's functions, instead of just a global on/off.

**Comment by Lunatrius** [comment]
Permission nodes?

**Comment by damonmensch** [comment]
It's a concept from back in bucket. Basically if the person is granted the node in a permissions based plugin our mod, then he or she can use the command freely, else they are denied access to using it. By default anyone opped has access to any and all command based permission nodes.

It allows server owners/admins to be able to grant aces to certain commands/features on a case by case basis when it comes to players.

Basically it takes your server-wide enabling of features and makes it player specific.

**Comment by Lunatrius** [comment]
I see. I do have something similar planned, but it's not a high priority at the moment.

And it should be already possible to disable the client side features for certain people (with personalized MOTDs), assuming a mod exists that allows that.

**Comment by Lunatrius** [comment]
Partially implemented by [SchematicaPlugin](https://github.com/Lunatrius/SchematicaPlugin).

**Comment by Lunatrius** [comment]
I just realized that the plugin does exactly what you wanted. I may add additional permissions in the future (if needed).
